go/worker/event/lambdaServer.go

```diff
@@ -49,19 +49,23 @@ func (s *LambdaServer) RunLambda(w http.ResponseWriter, r *http.Request) {
 	// components represent run[0]/<name_of_sandbox>[1]/<extra_things>...
 	// ergo we want [1] for name of sandbox
 	urlParts := getURLComponents(r)
+	w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
Do we want to set headers on the non-error path?  I figured a user's lambda function would set that.